test/kubemark/iks/startup.sh

```diff
@@ -235,13 +235,18 @@ EOF
   fi
   proxy_mem_per_node=50
   proxy_mem=$((100 * 1024 + proxy_mem_per_node*NUM_NODES))
+  hollow_kubelet_params=$(eval "for param in ${HOLLOW_KUBELET_TEST_ARGS:-}; do echo -n \\\"\$param\\\",; done")
+  hollow_kubelet_params=${hollow_kubelet_params%?}
+  hollow_proxy_params=$(eval "for param in ${HOLLOW_PROXY_TEST_ARGS:-}; do echo -n \\\"\$param\\\",; done")
+  hollow_proxy_params=${hollow_proxy_params%?}
```

Comment:
Why do we need to change this? This is not exercised in presubmits, have you tested kubemark on iks manually for this?

Reply:
Sorry, I didn't test this, I think this is similar to what we do in start-kubemark.sh. The HOLLOW_KUBELET_TEST_ARGS could contain multiple arguments, eg "--v=4 --port=8000", and we can't simply replace {{hollow_kubelet_params}} with HOLLOW_KUBELET_TEST_ARGS, because the pod command arguments should be separated.

Eg:
If the content of hollow-node_template.yaml is
```
command:
        - /kubemark
        - {{hollow_kubelet_params}}
```

then
```
command:
        - /kubemark
        - --v=4 --port=8000
```
won't work.

we need to convert it to:
```
command:
        - /kubemark
        - --v=4 
        - --port=8000
```

There is a similar logic in `cluster/gce/gci/configure-helper.sh`

Reply:
I agree - keeping it consistent looks fine.